### DISCUSSION TITLE
Anyone Implemented Tracing or hook into the Logging for Gotenberg?

### DISCUSSION THREAD

**Discussion by mariosaraiva** [gotenberg/gotenberg#670] [Q&A]
We have several services calling our Gotenberg instance. We want to trace requests. Our trace id is provided in the request headers. Is there any way we can hook into Gotenberg logging to add the trace id to the log messages? Or has anyone found a way to customize logging in Gotenberg? Or another work around for tracing?

**Comment by gulien** [comment]
The request id (under the logging field `trace`) is already available in the logs.
